casper/plot_tool.py

```diff
@@ -194,7 +199,8 @@ def make_thumbnails(self, frame_count_limit=IMAGE_LIMIT, xsize=1000, ysize=1000)
         iterator = 0
         for image in images:
             image.thumbnail(size, Image.ANTIALIAS)
-            image.save(self.thumbnail_path + str(1000 + iterator) + "thumbnail.png", "PNG")
+            image.save(self.thumbnail_path + str(1000 +
+                                                 iterator) + "thumbnail.png", "PNG")
```

Comment:
I would prefer
```python
image.save(self.thumbnail_path + str(1000 + iterator) +
           "thumbnail.png", "PNG")
```

Reply:
definitely, I'll go through and fix. Thanks!